addons/mrp/tests/test_order.py

```diff
@@ -4916,6 +4916,50 @@ def test_mo_without_resource_calendar(self):
         self.assertEqual(mo.state, 'done')
         self.assertEqual(mo.workorder_ids[0].duration_expected, 1440.0)
 
+    def test_plan_depending_workorder(self):
+        """
+        Checks that modifying the real duration of a workorder depending on other workorders that have already been started works.
+        """
+        # create a production with two workorders, one depending on the other
+        self.env.user.groups_id += self.env.ref('mrp.group_mrp_routings')
+        self.env.user.groups_id -= self.env.ref('mrp.group_mrp_workorder_dependencies')
+
+        self.product_depends = self.env['product.product'].create({
+            'name': 'depends',
+        })
+
+        self.bom_depends = self.env['mrp.bom'].create({
```

Comment:
```suggestion
        bom_depends = self.env['mrp.bom'].create({
```

Even more, rather than re-creating `self.bom_3`, why don't you just use it directly, after adding the `allow_operation_dependencies` and operation link that you need?
That would shorten the test and avoid creating unnecessary records. :slightly_smiling_face: